src/rules/at-conditional-rule-no-parentheses/index.js

```diff
@@ -0,0 +1,69 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-conditional-rule-no-parentheses");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Do not use () to surround statements for @-rules"
+});
+
+// postcss picks up else-if as else.
+const conditional_rules = ["if", "while", "else"];
+
+function report(atrule, result) {
+  utils.report({
+    message: messages.rejected,
+    node: atrule,
+    result,
+    ruleName
+  });
+}
+
+function fix(atrule) {
+  const regex = /(if)? ?\((.*)\)/;
+
+  // 2 regex groups: 'if ' and cond.
+  const groups = atrule.params.match(regex).slice(1);
+
+  atrule.params = Array.from(new Set(groups)).join(" ");
```

Comment:
I wonder if `Array.from` and `Set` are supported in the environments that stylelint supports?

Reply:
I'm not entirely sure. It looks like stylelint-scss has lodash bundled, so it's probably a better idea to use that to avoid compatibility issues.